client/common/login_data.js

```diff
@@ -0,0 +1,56 @@
+
+(function() {
+    'use strict';
+
+    angular.module('closeness')
+            .service('fb_Data', fb_data);
+    function fb_data($state) {
+        var self = this;
+        self.signup = function(data) {
+            console.log(data);
+
+            Accounts.createUser({
+                email: data.email,
+                password: data.id,
+                profile: {
+                    social_id: data.id,
+                    name: data.name,
+                    picture: data.pic,
+                    friends: data.friends,
+                    created_at: new Date()
+                }
+            }, function(error, result) {
+                if (error) {
+                    console.log(error);
+                    if (error.reason == "Email already exists.") {
+
+                    }
+                }
+                else {
+                    console.log('success');
+                }
+
+
+            });
+        };
+        self.update = function(data) {
+            Meteor.users.update(Meteor.user()._id, {$set: {'profile.friends': data.friends}});
```

Comment:
this should not be here. 
because we have many extra fields in user collection like geo, friends etc etc even if they are empty

we need to add more fields in user tables 

and hence need to create a publish/subscribe in that 

http://docs.meteor.com/#/full/meteor_users

same as this